src/scalar.jl

```diff
@@ -218,6 +356,52 @@ const asℝ = as(Real, -∞, ∞)
 
 const as_real = asℝ
 
+# Fallback method: print all transforms in order
+function Base.show(io::IO, ct::CompositeScalarTransform)
+    str = string(ct.transforms[1])
+    for ti in ct.transforms[begin+1:end]
+        str *= " ∘ "*string(ti)
+    end
+    print(io, str)
+end
+
+# If equivalent to asℝ₊, print as such. Two ways to achieve this
+function Base.show(io::IO, ct::CompositeScalarTransform{Tuple{TVShift{T}, TVExp}}) where T
+    if ct[1].shift == 0
+        print(io, "asℝ₊")
+    else
+        print(io, "as(Real, ", ct[1].shift, ", ∞)")
+    end
+end
+Base.show(io::IO, ct::CompositeScalarTransform{Tuple{TVExp}}) = print(io, "asℝ₊")
+
+# If equivalent to asℝ₋, print as such. Two ways to achieve this
+function Base.show(io::IO, ct::CompositeScalarTransform{Tuple{TVShift{T}, TVNeg, TVExp}}) where T
+    if ct[1].shift == 0
+        print(io, "asℝ₋")
+    else
+        print(io, "as(Real, -∞, ", ct[1].shift, ")")
+    end
+end
+Base.show(io::IO, ct::CompositeScalarTransform{Tuple{TVNeg, TVExp}}) = print(io, "asℝ₋")
+
+# If equivalent to as𝕀, print as such. Two ways to achieve this
+function Base.show(io::IO, ct::CompositeScalarTransform{Tuple{TVShift{T1}, TVScale{T2}, TVLogistic}}) where {T1, T2}
+    if ct[1].shift == 0 && ct[2].scale == 1
+        print(io, "as𝕀")
+    else
+        print(io, "as(Real, ", ct[1].shift, ", ", ct[1].shift + ct[2].scale, ")")
+    end
+end
```

Comment:
Also here, it's not the same as asI, so it shouldn't be printed in the same way.

Reply:
Fair point